llm-enhanced-local-assist-blueprint/mass_llm_enhanced_assist_blueprint_nlyaml

```diff
@@ -0,0 +1,419 @@
+blueprint:
+  domain: automation
+  name: Music Assistant - Local LLM Enhanced Voice Support Blueprint
+  source_url: https://github.com/music-assistant/voice-support/blob/main/local-assist-blueprint/mass_llm_enhanced_assist_blueprint_en.yaml
+  description: '
+    ![Image](https://github.com/music-assistant/voice-support/blob/main/assets/music-assistant.png?raw=true)
+
+    # Media afspelen via Music Assistant met hulp van een LLM
+
+    ### Gebruik
+
+    Elke zin moet:
+
+    * beginnen met de woorden `Speel` of `Luister naar` gevolgd door een beschrijving 
+    van de media die je wil afspelen
+
+    * optioneel gevolgd door een area of Music Assistant mediaspeler waarop je de media
+    wil afspelen
+
+
+    #### Voorbeelden
+
+    ```
+
+    Speel de beste Pink Floyd nummers in de keuken
+
+    Luister naar Jagged Little Pill in de woonkamer
+
+    Luister naar het album Greatest Hits door James Taylor in de keuken
+
+    Speel het nummer New Years Day in de slaapkamer
+
+    Speel A Hard Days Night door Billy Joel in de slaapkamer
+
+    Luister naar de playlist Classic Rock in het kantoor
+
+    Luister naar BBC Radio 1 in de slaapkamer
+
+    Speel het album Classical Nights op de Slaapkamer Sonos Speaker
+
+    Luister naar de plaat Classical op de Slaapkamer Sonos Speaker
+
+    Speel liedjes van U2
+
+    Speel muziek van de componist van Oppenheimer
+
+    Speel dat album met op de voorkant de naakte baby die naar een bankbiljet zwemt
+
+    ```
+  '
+  homeassistant:
+    min_version: 2024.6.0
+  input:
+    llm_agent:
+      name: LLM converation agent
+      description: The LLM agent you want to use to process the query
+      selector:
+        entity:
+          filter:
+            domain: conversation
+    music_assistant_settings:
+      name: Intstellingen voor afspelen via Music Assistant
+      icon: mdi:music
+      description: Je kunt deze instellingen gebruiken om te bepalen hoe Music Assistant
+        de muziek afspeelt
+      input:
+        default_player:
+          name: Standaard mediaspeler
+          description: 'De standaar Music Assistant mediaspeler die gebruikt wordt in het
+            geval niet bepaald kan worden waar de muziek op afgespeeld wordt. Dus wanneer
+            er geen doel gespecificeerd wordt in het commando, en dit ook niet bepaald kan 
+            worden op basis van de voice satellite waarop het commando gegeven wordt.
+
+            Laat dit leeg in het geval je geen standaard mediaspeler wil gebruiken'
+          selector:
+            entity:
+              filter:
+              - integration: music_assistant
+                domain:
+                - media_player
+              multiple: false
+          default:
+        play_continuously:
+          name: Play Continuously
```

Comment:
Eigenlijk zou het fijn zijn om dit consistent Radio modus te noemen.
Ik vraag me nog steeds af of het wel in de scripts moet hoor want we hebben tegenwoordig ook de "dont stop the music" functie die vanzelf radio modus aanzet aan het eind van de queue

Reply:
Ik vind het prima om het er uit te halen hoor, ik heb je op mijn mediaspelers inderdaad ook aan staan

Reply:
Veranderd zoals besproken op Discord